src/test/resources/sftp/local/OutBound/status-20221101171156056.json

```diff
@@ -1 +1,19 @@
-Hello World 1
\ No newline at end of file
+{
```

Comment:
Addresses @NathanRussellValtech comments in my previous PR around having some valid values 🙏🏼